Lib/math/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""
+This module provides access to integer related mathematical functions.
```

Comment:
`math` is float-related / `libm`?:
```suggestion
This module provides access to mathematical functions.
```

Reply:
Copy/paste mistake :-( Fixed.